generators/server/templates/src/main/java/package/security/PersistentTokenRememberMeServices.java.ejs

```diff
@@ -180,6 +180,10 @@ public class PersistentTokenRememberMeServices extends
      * <p>
      * The standard Spring Security implementations are too basic: they invalidate all tokens for the
      * current user, so when he logs out from one browser, all his other sessions are destroyed.
+     *
+     * @param request the request.
```

Comment:
I'm not a big fan of "response the response", I'd rather have no Javadoc here, as it doesn't help at all

Reply:
OK let's keep this for consistency with the other methods, as it's the only one in this case

Reply:
@jdubois Actually me neither. Its mainly to keep the javadoc tool happy :)